provisioning_templates/snippet/rancheros_cloudconfig.erb

```diff
@@ -0,0 +1,42 @@
+#cloud-config
+<%#
+kind: snippet
+name: rancheros_cloudconfig
+-%>
+hostname: <%= @host.shortname %>
+rancher:
+  network:
+    dns:
+      nameservers:
+        - <%= @host.subnet.dns_primary %>
+        - <%= @host.subnet.dns_secondary %>
+      search:
+        - <%= @host.domain %>
+    interfaces:
+      eth*:
+        dhcp: false
+      eth0:
+        address: <%= @host.ip -%>/<%= @host.subnet.cidr %>
+        gateway: <%= @host.subnet.gateway %>
```

Comment:
While this might be okay for your environment, our default templates should work with DHCP out of box. Can you have DHCP by default and only unless `@host.subnet.dhcp_boot_mode?` then do static configuration? You can inspire in our other templates, we do this quite often (grep `boot_mode`). Thanks.